draft-ietf-quic-multipath.md

```diff
@@ -635,6 +635,9 @@ data packets, including 0-RTT packets, using the initial Connection ID with
 sequence number 0 after the handshake concluded.
 
 ACK_MP frames (defined in {{ack-mp-frame}}) can be returned on any path.
+If the ACK_MP is prefered to be sent on the same path as the acknowledged
+packet, it can be beneficial to bundle an MP_ACK frame
```

Comment:
```suggestion
packet ({{compute-rtt}}), it can be beneficial to bundle an MP_ACK frame
```
It might be a good idea to add reference to the section explaining why that might be a good strategy.